framework/cordova.gradle

```diff
@@ -78,19 +78,26 @@ int compareVersionList(Collection aParts, Collection bParts) {
     pairs.findResult(aParts.size()-bParts.size()) {it[0] - it[1] != 0 ? it[0] - it[1] : null}
 }
 
-// Compare two version strings, such as "19.0.0" and "18.1.1.0". If all matched
+// Compare two version strings, such as "19.0.0-rc1" and "18.1.1.0". If all matched
 // elements are identical, the longer version is the largest by this method.
 // Examples:
+//   "20.0.0-rc1" > "19.0.0"
 //   "19.0.0" > "19"
 //   "19.0.1" > "19.0.0"
 //   "19.1.0" > "19.0.1"
 //   "19" > "18.999.999"
 int compareVersions(String a, String b) {
+    a = removeTagOfReleaseLifeCycle(a)
+    b = removeTagOfReleaseLifeCycle(b)
```

Comment:
Thanks for the review.

I thought that Android SDK Manager will change the version string by removing the pre-release identifier after the release so we don't need to compare with pre-release identifiers. But some users may use all versions of build-tools for debugging.

I don't know that the 3rd party libraries can be used for the solution. Is it acceptable for the project? It may cause vulnerability and reduce consistency.

Reply:
We use tons of 3rd party libraries on the Node.js side of our tools, so I don't see why we should not use Java/Groovy libraries. Especially when it's only a compile time dependency. I'm always happy when we can reduce the amount of code we have to maintain.

Reply:
Ok, thanks a lot. I am going to use the library.

Reply:
Nice! Thanks for giving this another go.